packages/wdio-sync/webdriverio-core.d.ts

```diff
@@ -572,6 +572,7 @@ declare namespace WebdriverIO {
     type MockFilterOptions = {
         method?: string | ((method: string) => boolean),
         headers?: Record<string, string> | ((headers: Record<string, string>) => boolean),
+        requestHeaders?: Record<string, string> | ((headers: Record<string, string>) => boolean),
```

Comment:
I see changes in webdriverio d ts, but why don't you modify the webdriverio tpl d ts file?

Reply:
Is it because someone forgot to generate these files?

Reply:
Yes, seems like it.